src/Components/FormikForm.js

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import { useFormik } from "formik";
+import { CREATE_POST_INPUT} from "../GraphQL/Mutations";
+import { useMutation } from "@apollo/client";
+const validate = values => {
+    const errors ={};
+
+    if (!values.title) {
+        errors.title = 'Required';
+    } else if (values.title.length > 15) {
+        errors.title = 'Must be 15 characters or less'
+    } else if (/^[а-яА-Я]+$/g.test(values.title)){
+        errors.title = 'Only English letter is allowed'
+    }
+
+    if(!values.body){
+        errors.body = 'Required';
+    } else if (values.body.length < 20){
+        errors.body = 'Must be 20 characters or more'
+    } else if (/^[а-яА-Я]+$/g.test(values.body)){
+        errors.body = 'Only English letter is allowed'
+    }
+    return errors;
+}
+function FormikForm ({ setOpenModal }) {
+    const [createPost, { error }] = useMutation(CREATE_POST_INPUT)
+
+    const formik = useFormik({
+        initialValues: {
+            title: '',
+            body: '',
+        },
+        validate,
+        onSubmit: values => {
+            createPost({
+                variables: {
+                    input: values
+                }
+            });
+            if (error){
+                console.log(error)
+            }
+        }
+    })
+    return(
+        <form onSubmit={formik.handleSubmit}>
+           <input
```

Comment:
To avoid describing so many properties for input I'd probably use formik `Field` or something like this `formik.getFieldProps`.